.travis.yml

```diff
@@ -0,0 +1,35 @@
+matrix:
+  include:
+    - language: python
+      python: 3.6
+      before_install:
+        - cd server/
+      install:
+        - pip install -r requirements.txt
+      script:
+        - python -m py_compile main.py
+
+    - language: c
+      before_script:
+        - cd arduino/
+      before_install:
+        - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+      install:
+        - arduino --install-library "Smartcar shield"
+      script:
+        - build_platform mega2560
+      cache:
+        directories:
+          - ~/arduino_ide
+          - ~/.arduino15/packages/
+
+deploy:
+  provider: releases
+  api_key:
+    secure: bdeac7bd7ffa0b46a10afbc198cdd6254d2cc6d1
+  file:
+    - /arduino/testCar/testCar.ino.with_bootloader.mega.hex
+    - /arduino/testCar/testCar.ino.mega.hex
```

Comment:
These two paths look wrong. You should probably add `$TRAVIS_BUILD_DIR` in front of them.